modules/ProspectLists/ProspectList.php

```diff
@@ -427,7 +427,7 @@ public function save($check_notify = false)
 
     public function mark_deleted($id)
     {
-        $query = "UPDATE prospect_lists_prospects SET deleted = 1, date_modified = NOW() WHERE prospect_list_id = '{$id}' AND deleted = 0";
+        $query = "UPDATE prospect_lists_prospects SET deleted = 1, date_modified = UTC_TIMESTAMP() WHERE prospect_list_id = '{$id}' AND deleted = 0";
```

Comment:
I don't believe this will work with MS SQL, so you will likely need to use the [DB convert](https://github.com/salesagility/SuiteCRM/blob/5f8062ceca2cf4d1c365ba777e3080be8604e4e5/include/database/DBManager.php#L3814) function here to support both engines and equally update that to use `UTC_TIMESTAMP` in MySQL , so this fix is applied globally when used